src/doc/gui_manual/Operators/OperatorTypes/ICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,156 @@
+    *Delete All Points*, the user can *Read Text File* that is formatted with
+    one point per each line either as "X Y Z" or "X, Y, Z".
+   
+Selection
+    Seed with a named selection.
+    
+Field Data
+    The seed points are defined by another operator and passed to the Integral
+    Curve operator. The array containing the seed points must begin its name
+    with "Seed Points".
+
+Up Axis
+    The "up axis" serves as the "Y" axis embedded in the plane or circle.
+
+Sampling type
+"""""""""""""
+
+For samples taken from a geometric object, there is an option to generate
+uniform or random samples from the specified region. Random samples can be
+reproduced by supplying a random number seed.
+
+Boundary vs Interior Samples
+""""""""""""""""""""""""""""
+
+Samples from a geometric object can be taken either from the boundary or the
+interior. For example, when sampling a plane, the samples can
+either lie along the edges of the planar region or within the bounded
+rectangle, as shown below.
+
+.. figure:: images/boundary_interior_sampling.png
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for numerical integrators. In addition to the
+:ref:`common ICS integration` attributes common to all ICS operators, the
+Integral Curve operator supports the following attributes.
+
+Integration Direction
+"""""""""""""""""""""
+
+Sets the integration direction through time. The user can choose from a
+combination of forward, backward, and directionless. Eigen vectors are an
+example of a directionless vector field. In order to integrate using a
+directionless field, any orientation discontinuity must be corrected prior to
+linear interpolation. That is, all vectors must be rotated to match the
+orientation of the trajectory. The ICS code will do this processing for
+standard fields (e.g non-higher order elements).
+
+Termination
+"""""""""""
+
+Integral curve termination can be controlled in several different ways. The
+termination is based on the most conservative criteria, so only one criteria
+must be met for termination. The options are:
+
+Maximum number of steps
+    The maximum number of integration steps that will be allowed. 
+
+Appearance
+^^^^^^^^^^
+
+The appearance tab specifies how the integral curve will be rendered. In
+addition to the :ref:`common ICS appearance` attributes common to all ICS
+operators, the Integral Curve operator supports the following attributes:
+
+Data
+""""
+
+The data type controls how the integral curves are colored. There are various
+options, the names of which are self-descriptive such as coloring the curves
+a *solid* color or according to a *seed*. Only those options that require
+further clarification are described further here.
+
+Average Distance from seed
+    Each curve is colored according to the average distance of all the points
+    in the curve from the seed.
+
+Variable
+    Each curve's color varies by the value of a scalar variable.
```

Comment:
Would prefer consistency in how these descriptions begin using either `Each curve is colored...` or `Each curve's color...`